ios/FluentUI/Navigation/Shy Header/ShyHeaderController.swift

```diff
@@ -66,11 +66,16 @@ class ShyHeaderController: UIViewController {
     private var contentScrollViewObservation: NSKeyValueObservation?
     private var previousContentScrollViewTraits = ContentScrollViewTraits() //properties of the scroll view at the last scrollDidOccurIn: update. Used with current traits to understand user action
 
-    init(contentViewController: UIViewController) {
+    // The context of the parent controller used to pull the correct FluentTheme to update visuals
+    weak var containingView: UIView?
```

Comment:
just for my understanding, do we need the entire containingView reference? or just the fluenttheme reference from navigationcontroller?

Reply:
Passing a fluentTheme reference is not guaranteed to work. If for whatever reason the parent view doesn't have a custom theme when we are grabbing its theme, it will return a reference to the shared theme and we will be stuck with that even when the parent view's theme changes.